proposals/lip-0045.md

```diff
@@ -1411,15 +1411,41 @@ def getTerminatedOutboxAccount(chainID: ChainID) -> TerminatedOutboxAccount:
     return terminatedOutboxAccount(chainID)
 ```
 
+### Endpoints for Off-Chain Services
+
 #### isChainIDAvailable
 
 ```python
 def isChainIDAvailable(chainID: ChainID) -> bool:
-    if chainAccount(chainID) does not exist:
-        return True
-    return False
+    if chainAccount(chainID) exist:
```

Comment:
should we also have some validation here like the chainID has correct length and format

Reply:
I think in the pseudocode we use the typing to assume implicitly that these checks are valid

Reply:
not only typing but length checks are missing here

Reply:
What I meant is that in the constant table we already say that type `ChainID` has length 4.

Reply:
@mitsuaki-u Please let me know if you agree with the reply from alessandro.

Reply:
I think it would be nice to also have length check in pseudo code as I recall seeing it in other pseudo codes but I defer to @ricott1 's decision

Reply:
I don't have a strong opinion, for consistency I wouldn't add it. In the implementation do you use a type for `ChainID` or do you check the length? Maybe we can go with what you do there